src/qibocal/auto/execute.py

```diff
@@ -34,7 +38,7 @@ class Executor:
     def load(
         cls,
         card: Runcard,
-        output: Path,
+        output: Path = None,
```

Comment:
Either
```suggestion
        output: Path,
```
or
```suggestion
        output: Optional[Path] = None,
```
(and the same would be true for the following two arguments).

Why do you need the default now? Nothing against that, just to understand.